### ISSUE TITLE
Incorrect Django requirements on readme

### ISSUE THREAD

**Issue by dtr5** [cfedermann/Appraise#51]
Readme states Appraise should work with Django >1.3, but it does not work with any 1.3 version, because certain symbols imported on urls.py were on *django.conf.urls.default* on 1.3, but on *django.conf.urls* on >1.4. 

Also, it does not work with >1.5, because the *patterns* api changed. There is also an open issue with 1.6.

To my best knowledge, it only works with **1.4.20**, that will be supported until October 2015. 

Can you update the readme to reflect this?

**Comment by housamz** [comment]
I agree, after many testings, Appraise only works perfectly django 1.4.2

**Comment by laeubli** [comment]
Confirming @dtr5 and @housamz's comments.

**Comment by cfedermann** [comment]
Appraise is definitely running on a Django 1.4.20 version.

I'm currently finishing the Django 1.9.6 version which will should be ready in the coming weeks. That will be the best way forward :)

**Comment by jmcoursi** [comment]
Hello,
Any news on the next version ? I installed on Django 1.4.20 but still cannot access the page to add new evaluation tasks. http://127.0.0.1:8000/appraise/admin/evaluation/evaluationtask/add/ 
Thanks !

**Comment by mbiesialska** [comment]
I've downloaded the current version, and the problem reported above by @jmcoursi persists.
According to the `readme` file
> **Add users**
Users can be added [here](http://127.0.0.1:8000/appraise/admin/auth/user/add/).

Causes the following error:
![image](https://user-images.githubusercontent.com/7369819/83896708-df14df80-a754-11ea-8f84-a42ede9b3a84.png)

>**Add evaluation tasks**
Evaluation tasks can be created [here](http://127.0.0.1:8000/appraise/admin/evaluation/evaluationtask/add/).

Causes the following error:
![image](https://user-images.githubusercontent.com/7369819/83897007-4c287500-a755-11ea-9548-b2e9af0a09e8.png)

It seems that templates for admin don't exist. 
Does anyone know how to solve this issue?

**Comment by SamuelLarkin** [comment]
Hi all,
  For future users that stumble on this issue.  Reading from stackoverflow, I found this https://stackoverflow.com/a/55101589.  After doing `pip install --no-cache-dir --no-binary Django django==1.4.22`, I'm now able to see the add/user & add/task pages.
